setup/advanced/kubernetes-sidecar.md

```diff
@@ -0,0 +1,119 @@
+# Kubernetes Sidecar Container Setup
+
+To get started with a Kubernetes sidecar container setup, simply adapt the following deployment yaml to suit your needs. Be sure to create a persistent volume and persistent volume claim for Gluetun to be able to store data!
+
+```yaml
+---
+ apiVersion: apps/v1
+ kind: Deployment
+ metadata:
+   name: gluetun-deployment
+   labels:
+     app: gluetun
+ spec:
+   replicas: 1
+   strategy:
+       type: RollingUpdate
+       rollingUpdate:
+          maxSurge: 1
+   selector:
+      matchLabels:
+        app: gluetun
+   template:
+     metadata:
+       labels:
+         app: gluetun
+     spec:
+      containers:
+         - name: gluetun
+           image: ghcr.io/qdm12/gluetun # Optionally you can use the "qmcgaw/gluetun" image as well as specify what version of Gluetun you desire
+           imagePullPolicy: Always
+           securityContext:
+             capabilities:
+               add: ["NET_ADMIN"]
+           # You can optionally specify ports that are used by your containers, but it is not strictly necessary
+           # ports:
+             # - containerPort: 8080
+
+           # Adapt the following environment variables to suit your needs and VPN provider
+           env:
+           - name: TZ
+             value: "Europe/London"
+
+           - name: VPN_SERVICE_PROVIDER
+             value: "ivpn"
+
+           - name: OPENVPN_USER
+             value: ""
+
+           - name: OPENVPN_PASSWORD
+             value: ""
+
+           # If you having connection issues, try enabling these variables to help diagnose it.
+           # - name: FIREWALL_DEBUG
+           #   value: "on"
+           # - name: FIREWALL_INPUT_PORTS
+           #   value: "8080"
```

Comment:
Can you set these to the same order/values + comments as the readme.md of gluetun:

```yml
      # See https://github.com/qdm12/gluetun-wiki/tree/main/setup#setup
      - VPN_SERVICE_PROVIDER=ivpn
      - VPN_TYPE=openvpn
      # OpenVPN:
      - OPENVPN_USER=
      - OPENVPN_PASSWORD=
      # Wireguard:
      # - WIREGUARD_PRIVATE_KEY=wOEI9rqqbDwnN8/Bpp22sVz48T71vJ4fYmFWujulwUU=
      # - WIREGUARD_ADDRESSES=10.64.222.21/32
      # Timezone for accurate log times
      - TZ=
      # Server list updater
      # See https://github.com/qdm12/gluetun-wiki/blob/main/setup/servers.md#update-the-vpn-servers-list
      - UPDATER_PERIOD=
```